src/aws-cur/cost-report.ts

```diff
@@ -103,13 +103,17 @@ export class CostReport extends Construct {
   constructor(scope: Construct, id: string, props: CostReportProps) {
     super(scope, id);
 
+    const currentStack = Stack.of(this);
+
+    if (currentStack.region !== 'us-east-1') {
+      throw new Error('CUR as it is only available in the us-east-1 region');
```

Comment:
It may be user-friendly to output which region was specified by users. (And I tried to change the message a bit.)

```suggestion
      throw new Error(`The \`CostReport\` construct is only available in the us-east-1 region, got: ${currentStack.region} region`);
```